### DISCUSSION TITLE
TrackSom data

### DISCUSSION THREAD

**Discussion by therop397** [ImmuneDynamics/Spectre#144] [Q&A]
Prior to running TrackSom, I assume one would need to transform the data as per the basic Spectre workflow (Arcsinh), rather than running on raw FlowJo files? Sorry if this seems very simple, but can't seem to find an answer in the documentation.

Thank you!

**Comment by ghar1821** [comment]
Hi there,

Yes you will need to transform your data first. Sorry, I should've made this clear in TrackSOM repo. 

You can use either the arcsinh transformation in Spectre or the one in FlowJo (including the linear binning method). It doesn't matter as long as they make sense.

**Reply by therop397** [reply]
Thank you, I'll give that a go!
